src/prepareOutput.js

```diff
@@ -283,15 +274,16 @@ export const stepExtractors = {
     "index-field-selection-step": indexFieldSelectionExtractor,
 }
 
-export function prepareOutput(formState, usedSteps) {
+export function prepareOutput(formState) {
+    const path = JSON.parse(localStorage.getItem("path")) || [];
```

Comment:
don't agree, this should not come from local storage, this is supposed to be a pure function no context and no side effects